templates/assets/cloudformation/service-iam.yml

```
@@ -309,7 +309,14 @@ Resources:
             Condition:
               ArnEquals:
                 "ecs:cluster": !Sub arn:${AWS::Partition}:ecs:${AWS::Region}:${AWS::AccountId}:cluster/${Namespace}-environment-${EnvironmentName}
-
+          - Effect: Allow
+            Action: 
+            - iam:PassRole 
+            Resource: "*"
```

Can this be changed to restrict what role can be passed from the CW event?

```suggestion
            Resource: !GetAtt EcsTaskRole.Arn
```